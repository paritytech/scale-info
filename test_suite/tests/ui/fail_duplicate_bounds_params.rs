use scale_info::TypeInfo;
use core::marker::PhantomData;

#[derive(TypeInfo)]
#[scale_info(bounds(), bounds())]
struct A<T> {
    marker: PhantomData<T>,
}

fn main() {}