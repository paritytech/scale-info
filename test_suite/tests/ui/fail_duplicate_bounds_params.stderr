error: Duplicate `bounds` attributes
 --> $DIR/fail_duplicate_bounds_params.rs:5:1
  |
5 | #[scale_info(bounds(), bounds())]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
